staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -396,47 +396,52 @@ func validateConvertedObject(in, out *unstructured.Unstructured) error {
 
 // restoreObjectMeta copies metadata from original into converted, while preserving labels and annotations from converted.
 func restoreObjectMeta(original, converted *unstructured.Unstructured) error {
```

Comment:
did the test added exercise the path where SSA invoked this on an object without metadata, and the conversion returned annotations?

Reply:
no, it didn't. Added.